Sources/ArgumentParser/Parsing/Interactive.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  File.swift
+//
+//
+//  Created by Keith  on 2022/6/21.
+//
+
+extension CommandParser {
+  func canInteract(error: Error, split: inout SplitArguments) -> Bool {
+    if let error = error as? ParserError {
+      switch error {
+      case let .missingValueForOption(inputOrigin, name):
+        var input: String?
+        while input?.isEmpty ?? true {
+          print("Please enter value for '\(name.synopsisString)': ", terminator: "")
+          input = readLine() ?? nil
+        }
+        
+        let index = inputOrigin.elements.first!.baseIndex! + 1
+        split._elements.append(.init(value: .value(input!),
+                                     index: .init(inputIndex: .init(rawValue: index))))
+        return true
+        
+      default:
+        break
+      }
+    }
+    
+    return false
+  }
+  
+  func canInteract(error: Error, values: inout ParsedValues) -> Bool {
+    if let error = error as? ParserError {
+      switch error {
+      case let .noValue(forKey: key):
```

Comment:
Same note as above for this method 👍🏻

Reply:
There may be more than one case here in the future, so let's keep the `switch` for now.